PolyEngine/Engine/Src/InputWorldComponent.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <Vector.hpp>
+#include "ComponentBase.hpp"
+#include "KeyBindings.hpp"
+
+namespace Poly
+{
+	namespace InputSystem
+	{
+		class InputWorldComponent : public ComponentBase
+		{
+		friend void InputPhase(World*);
+		public:
+			bool GetPressed(eKey key)						 { return CurrKey[static_cast<int>(key)]; }
+			bool GetPressed(eKey key1, eKey key2)			 { return (CurrKey[static_cast<int>(key1)] && CurrKey[static_cast<int>(key2)]) ? true : false; }
```

Comment:
If we want a method that checks multiple keys, lets write it with an argument std::initializer_list<eKey>.